README.md

```diff
@@ -44,7 +44,7 @@ in a dedicated `composer.json` file in your project, for example in the
 
 ```console
 mkdir -p tools/php-cs-fixer
-composer require --working-dir=tools/php-cs-fixer friendsofphp/php-cs-fixer
+composer require --working-dir=tools/php-cs-fixer --dev friendsofphp/php-cs-fixer
```

Comment:
I think it's worth to put it in `dev` section if you use main `composer.json`, but if you install it in standalone folder dedicated only for the dev-tool/php-cs-fixer, maybe it;s not needed

Reply:
I wanted to write the same 🙂. With this approach it's not needed because it's already separated from prod dependencies.

Reply:
Thanks, I updated the description, mentioning both ways.

What is the rationale behind using a separate `composer.json`? It is not a popular approach, afaik.

Reply:
avoid incompatibility with other packages

Reply:
@13DaGGeR dev dependencies shouldn't block your production dependencies from being installed/updated. In real life sometimes it's the case - you can't install/update something because some dev tool does not allow it. Why should it? That's why using PHARs or separate sets of dependencies (`composer.json` in separate dir) can help you - dependencies are segregated and separated.